Using .Graph in template leads to "panic: runtime error: invalid memory address or nil pointer dereference"

I'm using the the following template -- shortened:

```
template default {
  subject = {{.Last.Status}}: {{.Alert.Vars.name}} on {{.Group.host}}.{{.Group.domain}}
  body = `<p><b>Name:</b> {{.Alert.Name}}
  <p>{{ .Graph .Alert.Vars.q }}
`
}
```

`$q` is set in my alerts like this -- shortened:

```
alert ceph.linux.disk.os_in_progress.too.high {
  macro = alert.default

  $q = q("sum:5m-sum:linux.disk.ios_in_progress{dev=*,host=node*}", "10m", "")
  $metric = last($q)
  $warnValue = 500
  $critValue = 600
  warn = $metric > $warnValue
  crit = $metric > $critValue
}
```

This used to work until the last release. No I see the following exception in bosun logs:

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4e52d5]

goroutine 135 [running]:
runtime.panic(0x966260, 0xdf81d3)
        /usr/local/go/src/pkg/runtime/panic.c:279 +0xf5
text/template.errRecover(0xc20b1e7618)
        /usr/local/go/src/pkg/text/template/exec.go:100 +0xb9
runtime.panic(0x966260, 0xdf81d3)
        /usr/local/go/src/pkg/runtime/panic.c:248 +0x18d

....

github.com/bosun-monitor/bosun/sched.(*Context).eval(0xc20a116f00, 0x84d4e0, 0xc20b98e640, 0xe40101, 0x3e8, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /go/src/github.com/bosun-monitor/bosun/sched/template.go:148 +0x715
github.com/bosun-monitor/bosun/sched.(*Context).graph(0xc20a116f00, 0x84d4e0, 0xc20b98e640, 0x425f01, 0x0, 0x0, 0x0, 0x0)
        /go/src/github.com/bosun-monitor/bosun/sched/template.go:209 +0x9f
github.com/bosun-monitor/bosun/sched.(*Context).Graph(0xc20a116f00, 0x84d4e0, 0xc20b98e640, 0x0, 0x0, 0x0, 0x0)
        /go/src/github.com/bosun-monitor/bosun/sched/template.go:239 +0x69
reflect.Value.call(0x9bd240, 0xc20a116f00, 0x0, 0x1738, 0x9d4970, 0x4, 0xc2097c21c0, 0x1, 0x1, 0x0, ...)
        /usr/local/go/src/pkg/reflect/value.go:563 +0x1210
reflect.Value.Call(0x9bd240, 0xc20a116f00, 0x0, 0x1738, 0xc2097c21c0, 0x1, 0x1, 0x0, 0x0, 0x0)
        /usr/local/go/src/pkg/reflect/value.go:411 +0xd7
text/template.(*state).evalCall(0xc20a117100, 0x9bd240, 0xc20a116f00, 0x0, 0x160, 0x9bd240, 0xc20a116f00, 0x0, 0x1738, 0x7ff1a6db9848, ...)
        /usr/local/go/src/pkg/text/template/exec.go:563 +0x9d4
text/template.(*state).evalField(0xc20a117100, 0x9bd240, 0xc20a116f00, 0x0, 0x160, 0xc209d3c989, 0x5, 0x7ff1a6db9848, 0xc2093ede00, 0xc20b6450a0, ...)
        /usr/local/go/src/pkg/text/template/exec.go:474 +0x40e
text/template.(*state).evalFieldChain(0xc20a117100, 0x9bd240, 0xc20a116f00, 0x0, 0x160, 0x9bd240, 0xc20a116f00, 0x0, 0x160, 0x7ff1a6db9848, ...)
        /usr/local/go/src/pkg/text/template/exec.go:445 +0x322
text/template.(*state).evalFieldNode(0xc20a117100, 0x9bd240, 0xc20a116f00, 0x0, 0x160, 0xc2093ede00, 0xc20b6450a0, 0x2, 0x2, 0x0, ...)
        /usr/local/go/src/pkg/text/template/exec.go:412 +0x219
text/template.(*state).evalCommand(0xc20a117100, 0x9bd240, 0xc20a116f00, 0x0, 0x160, 0xc2093edc50, 0x0, 0x0, 0x0, 0x0, ...)
        /usr/local/go/src/pkg/text/template/exec.go:354 +0xbeb
text/template.(*state).evalPipeline(0xc20a117100, 0x9bd240, 0xc20a116f00, 0x0, 0x160, 0xc208193a40, 0x0, 0x0, 0x0, 0x0)
        /usr/local/go/src/pkg/text/template/exec.go:332 +0x1be
text/template.(*state).walk(0xc20a117100, 0x9bd240, 0xc20a116f00, 0x0, 0x160, 0x7ff1a6db9890, 0xc2090e67c0)
        /usr/local/go/src/pkg/text/template/exec.go:167 +0x133
text/template.(*state).walk(0xc20a117100, 0x9bd240, 0xc20a116f00, 0x0, 0x160, 0x7ff1a44d0dd8, 0xc2093fd5f0)
        /usr/local/go/src/pkg/text/template/exec.go:175 +0x848
text/template.(*Template).Execute(0xc20ba1e7c0, 0x7ff1a6db9e40, 0xc20aee6c40, 0x9bd240, 0xc20a116f00, 0x0, 0x0)
        /usr/local/go/src/pkg/text/template/exec.go:155 +0x332
html/template.(*Template).Execute(0xc20b644f80, 0x7ff1a6db9e40, 0xc20aee6c40, 0x9bd240, 0xc20a116f00, 0x0, 0x0)
        /usr/local/go/src/pkg/html/template/template.go:73 +0xa7
github.com/bosun-monitor/bosun/sched.(*Schedule).ExecuteBody(0xc20bab8d00, 0x7ff1a6db9e40, 0xc20aee6c40, 0xc2093dd9b0, 0xc20818c700, 0xc20baa2c60, 0x0, 0x0, 0x0, 0x0, ...)
        /go/src/github.com/bosun-monitor/bosun/sched/template.go:114 +0x1d1
github.com/bosun-monitor/bosun/web.procRule(0x7ff1a41a5b90, 0xc20819da40, 0xc2096ee680, 0xc20818c700, 0xecc0fd978, 0x0, 0xdf5c80, 0xe2eb00, 0xc20b625a76, 0x0, ...)
        /go/src/github.com/bosun-monitor/bosun/web/expr.go:135 +0xdbd
github.com/bosun-monitor/bosun/web.func·004(0x7ff1a41a5b90, 0xc20819da40)
        /go/src/github.com/bosun-monitor/bosun/web/expr.go:255 +0x1dd
github.com/bosun-monitor/bosun/_third_party/github.com/MiniProfiler/go/miniprofiler.(*Timing).Step(0xc2088ce180, 0xc20b93a600, 0xa, 0xc209c1e690)
        /go/src/github.com/bosun-monitor/bosun/_third_party/github.com/MiniProfiler/go/miniprofiler/types.go:214 +0x113
github.com/bosun-monitor/bosun/_third_party/github.com/MiniProfiler/go/miniprofiler.(*Profile).Step(0xc20b8b0120, 0xc20b93a600, 0xa, 0xc209c1e690)
        /go/src/github.com/bosun-monitor/bosun/_third_party/github.com/MiniProfiler/go/miniprofiler/types.go:193 +0x55
github.com/bosun-monitor/bosun/web.func·005()
        /go/src/github.com/bosun-monitor/bosun/web/expr.go:258 +0x262
created by github.com/bosun-monitor/bosun/web.Rule
        /go/src/github.com/bosun-monitor/bosun/web/expr.go:263 +0x1693

goroutine 16 [select (no cases), 1 minutes]:
main.main()
        /go/src/github.com/bosun-monitor/bosun/main.go:118 +0xd9a

...
```

Any idea what this can be? If you need more details, please feel free to ask.

--- a/sched/template.go
+++ b/sched/template.go
@@ -195,7 +195,11 @@ func (c *Context) eval(v interface{}, filter bool, series bool, autods int) ([]*
 	if series && e.Root.Return() != parse.TYPE_SERIES {
 		return nil, "", fmt.Errorf("egraph: requires an expression that returns a series")
 	}
-	res, _, err := e.Execute(c.runHistory.Context, nil, c.AbnormalEvent().Time, autods, c.Alert.UnjoinedOK, c.schedule.Search, c.schedule.Lookups, c.schedule.Conf.AlertSquelched(c.Alert))
+	t := time.Now().UTC()
+	if c.AbnormalEvent() != nil {
+		t = c.AbnormalEvent().Time
+	}
+	res, _, err := e.Execute(c.runHistory.Context, nil, t, autods, c.Alert.UnjoinedOK, c.schedule.Search, c.schedule.Lookups, c.schedule.Conf.AlertSquelched(c.Alert))
 	if err != nil {
 		return nil, "", fmt.Errorf("%s: %v", v, err)
 	}

